src/backend/src/modules/auth/keycloak-user.model.ts

```diff
@@ -0,0 +1,25 @@
+export class KeycloakUser {
+  idir_username?: string;
+
+  email_verified: boolean;
+
+  name: string;
+
+  preferred_username: string;
+
+  display_name: string;
+
+  given_name: string;
+
+  family_name?: string;
+
+  email?: string;
+
+  idir_user_guid?: string;
+
+  sub?: string;
```

Comment:
I would maybe strip `sub`, and also whichever of `idir_username` and `idir_user_guid` we are not using, just to stop people accidentally using them